spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -130,8 +117,10 @@
 
           # On Verify, user changes response from "Yes,..." to
           # "No, I live at a different address", see submitted_values above
-          step.call
+          @result = step.call
+        end
 
+        it 'retains identity_doc_ attrs/value but removes addr attr in flow session' do
           # retains identity_doc_ attributes and values in flow session
```

Comment:
maybe we don't need this comment anymore? seems duplicative of the line above it
```suggestion
```